### ISSUE TITLE
[Bug]: Gog Galaxy error "Essential components missing"

### ISSUE THREAD

**Issue by Ciberbago** [Calinou/scoop-games#907]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

goggalaxy

### Expected/Current Behaviour

I installed the package, when I open it, it asks for admin permissions, I grant them, but then a window with an error opens, it says: "Essential components needed to start GOG Galaxy are missing or incorrectly configured. If this continues, please reinstall the application". When it should start normally.

I already have vcredist2015 and innounp installed, as required in the manifest. I'm lost here, I tried in 2 different pc with windows 11 pro and the same thing happens.

### Steps to Reproduce

```console
PS> scoop install goggalaxy

*Open Gog Galaxy

*Error presents
```

### Possible Solution

I'm guessing this is some kind of dependency issue but I'm not sure. But it's really weird because using the official installer this does not happen.

### Scoop and Buckets Version

```console
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'ciber' bucket:
6e19be2 (HEAD -> main, origin/main, origin/HEAD) Added install instructions

'extras' bucket:
617f18f86 (HEAD -> master, origin/master, origin/HEAD) yaffplayer: Update to version 0.6.1.0

'games' bucket:
13e2fc7c (HEAD -> master, origin/master, origin/HEAD) pcsx2-dev: Update to version 1.7.4491

'main' bucket:
85890b50f (HEAD -> master, origin/master, origin/HEAD) eksctl: Update to version 0.141.0

'nirsoft' bucket:
28db834 (HEAD -> master, origin/master, origin/HEAD) Excluded two pad files that require a password. They need to be updated manually.

'nonportable' bucket:
73c21ac (HEAD -> master, origin/master, origin/HEAD) burp-suite-pro-np: Update to version 2023.5

'sysinternals' bucket:
952e2ec (HEAD -> main, origin/main, origin/HEAD) sysinternals-suite@2023.03.30: Fix hash (Closes #11)

'versions' bucket:
1d65b3030 (HEAD -> master, origin/master, origin/HEAD) kdeconnect-nightly: Update to version 1520
```

### Scoop Config

```console
last_update           : 12/05/2023 07:11:40 a. m.
aria2-warning-enabled : False
aria2-options         : --check-certificate true
scoop_repo            : https://github.com/ScoopInstaller/Scoop
scoop_branch          : master
```

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.3.4
PSEdition                      Core
GitCommitId                    7.3.4
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by Ciberbago** [comment]
I understand, is weird, my alternative right now it's to install it with chocolatey, works well in there. Unfortunately, I cannot replace it with either Playnite or Heroic, I use galaxy because of the time played feature :(

**Comment by EstebanMunoz** [comment]
Hi guys, I'm facing the same issue. Since vcredist2015 is no longer in the extras bucket I installed the vcredist2022 version, but the problem is still there. Got curious so downloaded the installer from GOG Galaxy and it worked, even tough it is the same version installed from the games bucket (currently, 2.0.67.2).

**Comment by hylianux** [comment]
I am also having this problem, same steps as above... this issue has not been fixed.

**Comment by Calinou** [comment]
cc @superusercode

**Comment by EstebanMunoz** [comment]
I checked the installation done by Scoop when the flag `"innosetup": true` is set, and it seems to ignore at least one directory. The `setup.exe` of Gog Galaxy contains 4 folders, one of them named `{%ALLUSERSPROFILE}` which is the environment variable that usually points to the `ProgramData` folder. However, when the installation is done, this folder doesn't appear in the system.

I updated the manifest and simply used the `/VERYSILENT` switch on the installer and uninstaller. Now I can open the app without problems.
